src/data/products.ts

```diff
@@ -0,0 +1,345 @@
+    },
+    targetType: 'MALE',
+    rankType: 'MANY_RECEIVE',
+  },
+  {
+    id: 6,
+    name: 'BBQ 오리지널치킨+콜라1.25L',
+    imageURL:
+      'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 27000,
+      discountRate: 0,
+      sellingPrice: 27_000,
+    },
+    brandInfo: {
+      id: 2088,
+      name: 'BBQ',
+      imageURL:
+        'https://st.kakaocdn.net/product/gift/gift_brand/20220216170226_38ba26d8eedf450683200d6730757204.png',
+    },
+    targetType: 'MALE',
+    rankType: 'MANY_RECEIVE',
+  },
+  {
+    id: 7,
+    name: 'BBQ 오리지널치킨+콜라1.25L',
+    imageURL:
+      'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 27000,
+      discountRate: 0,
+      sellingPrice: 27_000,
+    },
+    brandInfo: {
+      id: 2088,
+      name: 'BBQ',
+      imageURL:
+        'https://st.kakaocdn.net/product/gift/gift_brand/20220216170226_38ba26d8eedf450683200d6730757204.png',
+    },
+    targetType: 'MALE',
+    rankType: 'MANY_RECEIVE',
+  },
+  {
+    id: 8,
+    name: 'BBQ 오리지널치킨+콜라1.25L',
+    imageURL:
+      'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 27000,
+      discountRate: 0,
+      sellingPrice: 27_000,
+    },
+    brandInfo: {
+      id: 2088,
+      name: 'BBQ',
+      imageURL:
+        'https://st.kakaocdn.net/product/gift/gift_brand/20220216170226_38ba26d8eedf450683200d6730757204.png',
+    },
+    targetType: 'MALE',
+    rankType: 'MANY_RECEIVE',
+  },
+  {
+    id: 9,
+    name: 'BBQ 오리지널치킨+콜라1.25L',
+    imageURL:
+      'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 27000,
+      discountRate: 0,
+      sellingPrice: 27_000,
+    },
+    brandInfo: {
+      id: 2088,
+      name: 'BBQ',
+      imageURL:
+        'https://st.kakaocdn.net/product/gift/gift_brand/20220216170226_38ba26d8eedf450683200d6730757204.png',
+    },
+    targetType: 'MALE',
+    rankType: 'MANY_RECEIVE',
+  },
+  {
+    id: 10,
+    name: 'BBQ 오리지널치킨+콜라1.25L',
+    imageURL:
+      'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 27000,
+      discountRate: 0,
+      sellingPrice: 27_000,
+    },
+    brandInfo: {
+      id: 2088,
+      name: 'BBQ',
+      imageURL:
+        'https://st.kakaocdn.net/product/gift/gift_brand/20220216170226_38ba26d8eedf450683200d6730757204.png',
+    },
+    targetType: 'MALE',
+    rankType: 'MANY_RECEIVE',
+  },
+  {
+    id: 11,
+    name: 'BBQ 오리지널치킨+콜라1.25L',
+    imageURL:
+      'https://st.kakaocdn.net/product/gift/product/20231030175450_53e90ee9708f45ffa45b3f7b4bc01c7c.jpg',
+    price: {
+      basicPrice: 27000,
+      discountRate: 0,
+      sellingPrice: 27_000,
```

Comment:
numeric seprator를 사용할 때도 일관되게 사용해주시면 좋을거 같아요
```suggestion
      basicPrice: 27_000,
      discountRate: 0,
      sellingPrice: 27_000,
```